src/queueUI/Line.java

```diff
@@ -0,0 +1,75 @@
+package queueUI;
+
+import javax.swing.*;
+
+
+public class Line {
+
+    private Node startQueue, endQueue;
+    String queue = "";
+
+    // create constructor
+    public Line() {
+        startQueue = null;
+        endQueue = null;
+    }
+
+    // method to know if the queue is empty.
+    public boolean EmptyQueue() {
```

Comment:
Methods that return a `boolean` are named as if they were a question.
For example in this case, the name should be: `isEmptyQueue`